app/users-todo/users-todo.controller.js

```diff
@@ -0,0 +1,8 @@
+angular.module('app.module')
+    .controller('usersCntr', ['$rootScope', '$scope', 'appCasheService', '$state', function ($rootScope, $scope, appCasheService, $state) {
```

Comment:
refactor second parameter in controller to use function instead of array of arguments

Reply:
done